controllers/controllers.js

```diff
@@ -0,0 +1,15 @@
+const { readTopics, readArticles } = require("../models/models");
+
+const getTopics = (req, res) => {
+  return readTopics().then((rows) => {
+    res.status(200).send({ topics: rows });
+  });
+};
+
+const getArticles = (req, res) => {
+  return readArticles().then((rows) => {
+    res.status(200).send({ articles: rows });
+  });
+};
```

Comment:
reachTopics and ReadArticles shouldnt need to be returned, and both of these should also have a .catch which invokes next just in case there are errors.

Reply:
returns removed and catch blocks added